src/status_im/ui/screens/signing/sheets.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im.ui.screens.signing.sheets
+  (:require-macros [status-im.utils.views :as views])
+  (:require [status-im.ui.components.react :as react]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.text-input.view :as text-input]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.button.view :as button]
+            [status-im.ui.components.colors :as colors]))
+
+(views/defview fee-bottom-sheet [fee-display-symbol]
+  (views/letsubs [{gas-edit :gas gas-price-edit :gasPrice max-fee :max-fee} [:signing/edit-fee]]
+    [react/view
```

Comment:
that level of nesting is really overkill, here it is no big deal because it's not a component in a list though